tox.ini

```diff
@@ -121,6 +121,10 @@ envlist =
     py3{5,6,7,8}-test-instrumentation-celery
     pypy3-test-instrumentation-celery
 
+    ; opentelemetry-instrumentation-sklearn
+    py3{5,6,7,8}-test-instrumentation-sklearn
+    pypy3-test-instrumentation-sklearn
```

Comment:
sklearn isn't available on pypy so you can remove this

Reply:
Awesome, thank you! @crflynn